decompiler/ObjectFile/ObjectFileDB.cpp

```diff
@@ -477,37 +478,37 @@ void ObjectFileDB::find_code() {
     if (get_config().game_version == 1 || obj.record.to_unique_name() != "effect-control-v0") {
       obj.linked_data.process_fp_relative_links();
     } else {
-      printf("skipping process_fp_relative_links in %s\n", obj.record.to_unique_name().c_str());
+      spdlog::warn("Skipping process_fp_relative_links in {}", obj.record.to_unique_name().c_str());
     }
 
     auto& obj_stats = obj.linked_data.stats;
     if (obj_stats.code_bytes / 4 > obj_stats.decoded_ops) {
-      printf("Failed to decode all in %s (%d / %d)\n", obj.record.to_unique_name().c_str(),
-             obj_stats.decoded_ops, obj_stats.code_bytes / 4);
+      spdlog::warn("Failed to decode all in {} ({} / {})", obj.record.to_unique_name().c_str(),
+                   obj_stats.decoded_ops, obj_stats.code_bytes / 4);
     }
     combined_stats.add(obj.linked_data.stats);
   });
 
-  printf("Found code:\n");
-  printf(" code %.3f MB\n", combined_stats.code_bytes / (float)(1 << 20));
-  printf(" data %.3f MB\n", combined_stats.data_bytes / (float)(1 << 20));
-  printf(" functions: %d\n", combined_stats.function_count);
-  printf(" fp uses resolved: %d / %d (%.3f %%)\n", combined_stats.n_fp_reg_use_resolved,
-         combined_stats.n_fp_reg_use,
-         100.f * (float)combined_stats.n_fp_reg_use_resolved / combined_stats.n_fp_reg_use);
+  spdlog::info("Found code:");
+  spdlog::info(" Code {} MB", combined_stats.code_bytes / (float)(1 << 20));
+  spdlog::info(" Data {} MB", combined_stats.data_bytes / (float)(1 << 20));
+  spdlog::info(" Functions: {}", combined_stats.function_count);
+  spdlog::info(" fp uses resolved: {} / {} ({} %%)", combined_stats.n_fp_reg_use_resolved,
```

Comment:
In spdlog we should just use a single `%` to get a percent sign.  The `printf` format needs `%%` to print a `%`.